frontend/src/lib/taxonomy.tsx

```diff
@@ -1096,7 +1096,7 @@ for (const [key, value] of Object.entries(CORE_FILTER_DEFINITIONS_BY_GROUP.event
     if (SESSION_INITIAL_PROPERTIES_ADAPTED_FROM_EVENTS.has(key)) {
         CORE_FILTER_DEFINITIONS_BY_GROUP.session_properties[`$initial_${key.replace(/^\$/, '')}`] = {
             ...value,
-            label: `Initial ${value.label}`,
+            label: `Session ${value.label}`,
```

Comment:
flyby to make sure searching will work if I just type "session" from now on, I don't think this works automatically

Reply:
Ah true - I worked on this for some other properties! Will fix